File: platform-instances/backend-for-frontend/src/main/java/com/sv/youapp/bff/BackForFrontEndApplication.java
package com.sv.youapp.bff;

import com.sv.youapp.bff.properties.BackEndForFrontEndProperties;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.security.reactive.ReactiveUserDetailsServiceAutoConfiguration;
import org.springframework.boot.context.properties.EnableConfigurationProperties;

@EnableConfigurationProperties({BackEndForFrontEndProperties.class})
@SpringBootApplication(exclude = {ReactiveUserDetailsServiceAutoConfiguration.class})
public class BackForFrontEndApplication {
  public static void main(String[] args) {
    SpringApplication.run(BackForFrontEndApplication.class, args);
  }
}

File: platform-instances/backend-for-frontend/src/main/kotlin/com/sv/youapp/bff/BackForFrontEndApplication.kt
package com.sv.youapp.bff

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.autoconfigure.security.reactive.ReactiveUserDetailsServiceAutoConfiguration
import org.springframework.boot.runApplication

@SpringBootApplication(    exclude = [
    ReactiveUserDetailsServiceAutoConfiguration::class
])
class BackForFrontEndApplication

fun main(args: Array<String>) {
    runApplication<BackForFrontEndApplication>(*args)
}
